### ISSUE TITLE
`TypeError: GuestTokenManager.reset() takes 1 positional argument but 2 were given`

### ISSUE THREAD

**Issue by garyrob** [JustAnotherArchivist/snscrape#885]
### Describe the bug

I get a blowup when trying to scrape for XplodingUnicorn

### How to reproduce

Here's a script which you can run:
```
import snscrape.modules.twitter as sntwitter
list(sntwitter.TwitterSearchScraper( "from:XplodingUnicorn -filter:retweets").get_items())
```

### Expected behaviour

It finishes successfully with no error.

### Screenshots and recordings

No screenshots or recordings but there is console output

### Operating system

macOS 13.3.1

### Python version: output of `python3 --version`

Python 3.10.3

### snscrape version: output of `snscrape --version`

snscrape 0.6.2.20230321.dev12+gea49c19

### Scraper

TwitterSearchScraper

### How are you using snscrape?

Module (`import snscrape.modules.something` in Python code)

### Backtrace

Here's the console output:

```
python test1.py
Unavailable user in card on tweet 1399697133670916096
Unavailable user in card on tweet 1399697133670916096
User 18899974 not found in user refs in card on tweet 1399697133670916096
User 18899974 not found in user refs in card on tweet 1399697133670916096
Empty user ref object in card on tweet 784137617273188358
User 899733078641225728 not found in user refs in card on tweet 784137617273188358
Traceback (most recent call last):
  File "/Users/garyrob/Source/yellowcow/TwitterApp/scripts/test1.py", line 4, in <module>
    list(sntwitter.TwitterSearchScraper( "from:XplodingUnicorn -filter:retweets").get_items())
  File "/Users/garyrob/.virtualenvs/yellowcow/lib/python3.10/site-packages/snscrape/modules/twitter.py", line 1704, in get_items
    for obj in self._iter_api_data('https://twitter.com/i/api/graphql/7jT5GT59P8IFjgxwqnEdQw/SearchTimeline', _TwitterAPIType.GRAPHQL, params, paginationParams, cursor = self._cursor, instructionsPath = ['data', 'search_by_raw_query', 'search_timeline', 'timeline', 'instructions']):
  File "/Users/garyrob/.virtualenvs/yellowcow/lib/python3.10/site-packages/snscrape/modules/twitter.py", line 812, in _iter_api_data
    obj = self._get_api_data(endpoint, apiType, reqParams, instructionsPath = instructionsPath)
  File "/Users/garyrob/.virtualenvs/yellowcow/lib/python3.10/site-packages/snscrape/modules/twitter.py", line 768, in _get_api_data
    r = self._get(endpoint, params = params, headers = self._apiHeaders, responseOkCallback = self._check_api_response)
  File "/Users/garyrob/.virtualenvs/yellowcow/lib/python3.10/site-packages/snscrape/base.py", line 266, in _get
    return self._request('GET', *args, **kwargs)
  File "/Users/garyrob/.virtualenvs/yellowcow/lib/python3.10/site-packages/snscrape/base.py", line 237, in _request
    success, msg = responseOkCallback(r)
  File "/Users/garyrob/.virtualenvs/yellowcow/lib/python3.10/site-packages/snscrape/modules/twitter.py", line 755, in _check_api_response
    self._unset_guest_token(blockUntil)
  File "/Users/garyrob/.virtualenvs/yellowcow/lib/python3.10/site-packages/snscrape/modules/twitter.py", line 745, in _unset_guest_token
    self._guestTokenManager.reset(blockUntil)
TypeError: GuestTokenManager.reset() takes 1 positional argument but 2 were given
```

### Log output

[error.log](https://github.com/JustAnotherArchivist/snscrape/files/11447612/error.log)

### Dump of locals

"(snscrape should tell you the path)" I don't see anything like that.

**Comment by JustAnotherArchivist** [comment]
Yep, I found this earlier as well and already fixed it in commit 786815dd.

**Comment by garyrob** [comment]
OK! I upgraded and it's working now, thanks!!!
